hackathon_site/dashboard/frontend/src/components/inventory/InventoryFilter/InventoryFilter.js

```diff
@@ -12,79 +12,74 @@ import FormControlLabel from "@material-ui/core/FormControlLabel";
 
```

Comment:
I can't comment on the line itself, but line 3 the `Paper` import can be removed